baseband/mark5b/header.py

```diff
@@ -259,39 +259,31 @@ def get_time(self, framerate=None, frame_nr=None):
         In the code, this is "unrounded" to give the exact time of the start
         of the frame for any total bit rate below 512 Mbps.  For rates above
         this value, it is no longer guaranteed that subsequent frames have
-        unique rates, and one should pass in an explicit frame rate instead.
+        unique frac_sec, and one should pass in an explicit frame rate instead.
```

Comment:
Seeing how often this is referenced, I wonder if it would be straightforward to replace `ns` with `frac_sec`?  I don't understand still how that "unround" line works, but otherwise `ns` is basically never used by anything.

Reply:
Also, frac_sec -> \`frac_sec\`.

Reply:
Will do the change to `fraction` and will then also adapt this.